go.mod

```diff
@@ -1,6 +1,6 @@
 module get.porter.sh/operator
 
-go 1.20
+go 1.21
```

Comment:
Because you are updating this, it’ll help if you also update controller-runtime to v0.17. This dep is also at go version 1.21